src/main/java/macnil/academy/controller/NewsRestController.java

```diff
@@ -42,7 +36,15 @@ public News read(@PathVariable ("id") Long id){
     }
 
 
-    }
-
     
 
+    @GetMapping
+    public  List <News> readAll1( Integer numero,@RequestParam( name="tenantId")  Long tenantId ){  //@RequestBody    required = false,
```

Comment:
readAll1?

il tenant non può essere un parametri di query (quindi opzionale) ma deve essere un path variable.
Stando alla naming convention rest https://restfulapi.net/resource-naming/ stiamo accedendo nell'albero delle risorse (che si viene a creare con le relazioni tra entità)
tenants -> news e non stiamo invece filtrando una collezione di risorse in base a un attivuto "le news dell'ultima settimana" qui serve un query param